lib/src/Pos/Launcher/Mode.hs

```diff
@@ -75,12 +77,17 @@ instance HasSlottingVar InitModeContext where
     slottingTimestamp = imcSlottingVar_L . _1
     slottingVar = imcSlottingVar_L . _2
 
+-- how would i embed this value inside the InitMode structure? (which i would have to turn into a data record?)
```

Comment:
`InitMode` is a typeclass so you can't.

Reply:
i think reflection and `Given` would be simpler, but i can see that being a pain to thread thru half of cardano...

Reply:
i have moved it into the `NodeDBs` structure, so it lives beside the paths of the db files and such